homepage/neuroimaging.rst

```diff
@@ -26,21 +26,32 @@ The technology and physics behind an MRI scanner is quite astonishing. But I won
        :width: 160pt
        :align: center
 
-The volume data of a whole brain recorded at one single timepoint, as pictured on the left, is called a **volume** and consists of multiple voxels (pixels in 3D). Each **voxel** has a specific dimension, in this case it's an isotropic dimension of 1mm x 1mm x 1mm. Each voxel contains one value which stands for the average signal measured at the given location. A standard anatomical volume, with an isotropic voxel resolution of 1mm contains almost 17 million voxels, which are arranged in a **3D matrix** of 256 x 256 x 256 voxels. The following picture describes a vertical slice through such a brain volume and depicts in general the grid like structure of this 3D Matrix, consisting of multiple voxels.
+The brain occupies space, so when we collect data on how it fills space, we call that volume data, and all the volume data needed to create the complete, 3D image of the brain, recorded at one single timepoint and as pictured on the left, is called a **volume**.  The data is measured in **voxels**, which are like the pixels used to display images on your screen, only in 3D. Each voxel has a specific dimension, in this case it is 1mm x 1mm x 1mm: a cube, so it is the same dimension from all sides (isotropic). Each voxel contains one value which stands for the average signal measured at the given location.
+
+A standard anatomical volume, with an isotropic voxel resolution of 1mm contains almost 17 million voxels, which are arranged in a **3D matrix** of 256 x 256 x 256 voxels. The following picture shows a vertical slice -- one layer of the big, 3D grid looked at from the top -- through a brain volume, and the grid for the remaining two dimensions is superimposed on the picture of the brain.  The data is measured per box, and each of the boxes corresponds to a particular portion of the brain.
 
 .. image:: images/voxel.png
    :width: 300pt
    :align: center
 
-As the scanner can't measure the whole volume at once it has to measure it sequentially. This is done by measuring one plane of the brain  (generally the horizontal one) after the other. Such a plane is also called a **slice**. The resolution of the measured volume data therefore depends on the in-plane resolution, the number of slices, their thickness, and a possible gap between them.
+As the scanner can't measure the whole volume at once it has to measure portions of the brain sequentially in time. This is done by measuring one plane of the brain (generally the horizontal one) after the other. Such a plane is also called a **slice**. The **resolution** of the measured volume data, therefore, depends on the in-plane resolution (the size of the squares in the above image), the number of slices and their thickness (how many layers), and any possible gaps between the layers.
 
```

Comment:
I removed the word vertical.  As I said in the comment, I like the grid
superimposition.  As you point out, though, it might be less disorienting
if the image were a horizontal slice.  If it's easy to contrive one with
the grid, then I'd say it's worth it.  Otherwise, I think it will be fine
so long as we don't say 'vertical' or 'horizontal' at all.

-- bennet


On Mon, Feb 22, 2016 at 7:21 AM, Michael Notter <notifications@github.com>
wrote:

> In homepage/neuroimaging.rst
> <https://github.com/miykael/nipype-beginner-s-guide/pull/8#discussion_r53617384>
> :
>
> >
> >  .. image:: images/voxel.png
> >     :width: 300pt
> >     :align: center
> >
> > -As the scanner can't measure the whole volume at once it has to measure it sequentially. This is done by measuring one plane of the brain  (generally the horizontal one) after the other. Such a plane is also called a **slice**. The resolution of the measured volume data therefore depends on the in-plane resolution, the number of slices, their thickness, and a possible gap between them.
> > +As the scanner can't measure the whole volume at once it has to measure portions of the brain sequentially in time. This is done by measuring one plane of the brain (generally the horizontal one) after the other. Such a plane is also called a **slice**. The **resolution** of the measured volume data, therefore, depends on the in-plane resolution (the size of the squares in the above image), the number of slices and their thickness (how many layers), and any possible gaps between the layers.
> >
>
> Perfect changes, thank you. And as you write, usually slices are recorded
> horizontally. But the picture above is actually vertically, which might be
> confusing. Should I recreate the image with a horizontal slice?
> For example something like this:
> http://www.neurology.org/content/57/9/1636/F1.large.jpg
> Or even something like this - but without the functional overlay:
> https://theinfomonkeydotcom.files.wordpress.com/2015/02/improv-fmri-slices.jpg
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/miykael/nipype-beginner-s-guide/pull/8/files#r53617384>
> .
>